build.wake

```diff
@@ -51,12 +51,18 @@ def bootstrapTarget targetFn variant =
     else Pass paths
 
     def cppSuffixRegex =
-        (".", variant.getPairFirst,)
```

Comment:
This should be made to end in `Nil` instead now, all these small cases should work that way